src/components/partner/partner.repository.ts

```diff
@@ -119,12 +120,34 @@ export class PartnerRepository extends DtoRepository<
           relation('out', '', 'languageOfWiderCommunication', ACTIVE),
           node('languageOfWiderCommunication', 'Language'),
         ])
+        .subQuery('node', (sub) =>
+          sub
+            .match([
+              node('node'),
+              relation('out', '', 'languagesOfConsulting', ACTIVE),
+              node('languagesOfConsulting', 'Language'),
+            ])
+            .return(
+              'collect(languagesOfConsulting.id) as languagesOfConsultingIds',
+            ),
+        )
+        .with([
+          'node',
+          'languagesOfConsultingIds',
+          'props',
+          'sensitivity',
+          'organization',
+          'pointOfContact',
+          'languageOfWiderCommunication',
+          'scopedRoles',
+        ])
```

Comment:
We do the subquery above to avoid having to do this.
```suggestion
```

Reply:
You're correct that this is not needed for the API response. However, without these definitions the e2e tests breaks with 'undefined var' errors. I'm definitely not a cypher expert but after a bit of research this 'with' solved those. Open to other ideas...

Reply:
What was the error exactly?